### PR TITLE
resize disk online

### PR DESCRIPTION
**Pull request by john-kheir**

### CODE REVIEW

File: functional_testing/Openvcloud/ovc_master_hosted/OVC/c_advanced/machine_tests.py [hunk 1]
```diff
@@ -744,11 +744,52 @@ def test015_check_IP_conflict(self):
...
         vm_1_client.execute("ip a a {} dev {}".format(cs_ip,ext_interface_name1), sudo=True)
         vm_1_client.execute("nohup bash -c 'ip l s dev {} up </dev/null >/dev/null 2>&1 & '".format(ext_interface_name1), sudo=True)
 
         self.lg("Check that you can't connect to VM1 with it's new ext network ip, should succeed")
         with self.assertRaises(socket.error):
             VMClient(vm1_id,ip=cs_ip, external_network=True, timeout=1)
         self.lg('%s ENDED' % self._testID)
+
+    def test016_resize_disk_online(self):
+        """ OVC-047
```
Comment (ahmedelsayed-93) [inline functional_testing/Openvcloud/ovc_master_hosted/OVC/c_advanced/machine_tests.py]: update testcase id

File: functional_testing/Openvcloud/ovc_master_hosted/OVC/c_advanced/machine_tests.py [hunk 2]
```diff
@@ -744,11 +744,52 @@ def test015_check_IP_conflict(self):
...
+        self.lg('Attach data disk (DS1) with size (S1), should succeed')
+        s1 = random.randint(10, 50)
+        disk_id = self.create_disk(self.account_id, size=s1)
+        self.assertTrue(disk_id)
+        response = self.api.cloudapi.machines.attachDisk(machineId=vm1_id, diskId=disk_id)
+        self.assertTrue(response)
+
+        self.lg('Resize DS1 to S2, should succeed')
+        self.api.cloudapi.disks.resize(diskId=disk_id, size=s1 + 1)
+
```
Comment (ahmedelsayed-93) [inline functional_testing/Openvcloud/ovc_master_hosted/OVC/c_advanced/machine_tests.py]: add step to try to increase disk size and it should fail

File: functional_testing/Openvcloud/ovc_master_hosted/OVC/c_advanced/machine_tests.py (hunk 1 above)
Comment (john-kheir) [reply functional_testing/Openvcloud/ovc_master_hosted/OVC/c_advanced/machine_tests.py]: done

File: functional_testing/Openvcloud/ovc_master_hosted/OVC/c_advanced/machine_tests.py (hunk 2 above)
Comment (john-kheir) [reply functional_testing/Openvcloud/ovc_master_hosted/OVC/c_advanced/machine_tests.py]: no need .. this api is tested before

**Pull request merged**